src/tools/glam.md

```diff
@@ -0,0 +1,60 @@
+# GLAM
+
+## What is GLAM?
+
+GLAM was built to help Mozillians answer their data questions without needing data analysis or coding skills. GLAM contains a visualization layer meant to answer most “easy” questions about how a probe/metric has changed over build ids and releases.
+
+### The differences between GLAM, GUD, and TMO
+
+There are 3 different telemetry visualization dashboards currently being maintained at Mozilla:
+
+- **[Mozilla Growth & Usage Dashboard (GUD)](https://gud.telemetry.mozilla.org/)** is focused on a relatively small number of high level, derived product metrics about user engagement (e.g. MAU, DAU, retention, etc.) 
+
+- **[Glean Aggregated Metrics Dashboard (GLAM)](https://glam.telemetry.mozilla.org/)** is focused on a diverse and plentiful set of probes and data points that engineers capture in code and transmit back from Firefox and other Mozilla products.
+
+- **[The Telemetry Measurement Dashboard (TMO)](https://telemetry.mozilla.org/)** is the predecessor to GLAM (see above) and is still maintained until GLAM is considered production ready.
```

Comment:
```suggestion
- **[The Telemetry Measurement Dashboard (TMO)](https://telemetry.mozilla.org/)** is the predecessor to GLAM (see above) and is still being lightly maintained until we are sure GLAM covers all of its use cases.
```

I think we now consider GLAM production ready. I'm not sure what the schedule is for taking down TMO: @robhudson does the above sound accurate?

Reply:
Yep that sounds right to me.